packages/core/build/tokens.ts

```diff
@@ -12,6 +12,7 @@ const layout = {
     xs: token(4),
     sm: token(6),
     md: token(12),
+    xmd: token(18),
```

Comment:
@bbogdanov : I'm going to have to block this for the time being while we get aligned with the UX folks over these spacing sizes. We just need to make sure we are all on the same page here so that we can move forward in the spirit of consistency and long-term stability here. 

Thankfully, this is not a breaking change. You've done well. Maybe we could do with naming this `xm` instead of `xmd` because all the other sizes are just two letters.

But there may be more sizes like this coming our way and we need to understand the scope and reasoning. Right now, we don't know if this will go back to `24px` like the padding in the modal and overlays or remain `18px` and introduce new sizes in different areas.